### PR TITLE
Add support for serializing objects/arrays

### PR DESCRIPTION
**Pull request by bkutil**
As core Juttle now handles fields whose values are arrays or objects,
add support for serializing those points into JSON.

### CODE REVIEW

**Comment by bkutil** [comment]
cc: @VladVega

refs: https://github.com/juttle/juttle/issues/55

File: lib/write.js [hunk 1]
```diff
@@ -48,8 +48,10 @@ var Write = Juttle.proc.sink.extend({
     formatPoints: function(points) {
         _.each(points, function(pt) {
             _.each(pt, function(value, key) {
-                if (value instanceof JuttleMoment) {
+                if (values.isDate(value) || values.isDuration(value)) {
                     pt[key] = new Date(value.valueOf());
+                } else if (values.isObject(value) || values.isArray(value)) {
+                    pt[key] = JSON.stringify(value);
```
Comment (dmajda) [inline lib/write.js]: Same question as for Influx: wouldn’t it be better to just reject these points?

Comment (bkutil) [inline lib/write.js]: And a slightly different answer :) Unlike Influx, I think it is a common scenario to store JSON into string columns of an SQL database (we actually did this in the original Jut frontend). I'd keep it here, for Influx, I'm still not sure :)

Comment (bkutil) [inline lib/write.js]: cc: @demmer

**Comment by dmajda** [comment]
+1

**Pull request merged**